fmralign/tests/test_pairwise_alignment.py

```diff
@@ -34,6 +43,26 @@ def test_pairwise_identity():
         assert_algo_transform_almost_exactly(
             algo, img1, img1, mask=mask_img)
 
+    # test intersection of clustering and mask
+    data_mask = copy.deepcopy(mask_img.get_data())
+    data_mask[0] = 0
+    # create ground truth
+    clustering_mask = new_img_like(mask_img, data_mask)
+    data_clust = copy.deepcopy(data_mask)
+    data_clust[1] = 3
+    # create 2-parcels clustering, smaller than background
+    clustering = new_img_like(mask_img, data_clust)
+
+    # clustering is smaller than mask
+    assert (mask_img.get_data() > 0).sum() > (clustering.get_data() > 0).sum()
+    algo = PairwiseAlignment(alignment_method='identity',
+                             mask=mask_img, clustering=clustering)
+    with pytest.warns(UserWarning):
+        algo.fit(img1, img1)
+    assert (algo.mask.get_data() > 0).sum() == (
+        clustering.get_data() > 0).sum()
+    algo.transform(img1)
```

Comment:
It doesn't seem like this transform is being tested ?

Reply:
I removed it, it's hard to further test